app/page.tsx

```
@@ -1,137 +1,74 @@
+        <ResponsiveContainer maxWidth="xl">
+          <h2 className="text-3xl font-bold tracking-tight mb-8">Browse Categories</h2>
+          <FeaturedCategories categories={featuredCategories} />
+        </ResponsiveContainer>
+      </section>
+
+      {/* Recent Providers */}
+      <section className="py-16 bg-muted/30">
+        <ResponsiveContainer maxWidth="xl">
+          <div className="flex flex-col md:flex-row md:items-center justify-between mb-8">
+            <h2 className="text-3xl font-bold tracking-tight">Recently Added Providers</h2>
+            <a href="/providers/list" className="text-primary hover:underline mt-2 md:mt-0">
```

ESLint

[error] Do not use an `<a>` element to navigate to `/providers/list/`. Use `<Link />` from `next/link` instead. See: https://nextjs.org/docs/messages/no-html-link-for-pages

(@next/next/no-html-link-for-pages)